pkg/services/live/live.go

```diff
@@ -724,6 +720,13 @@ func (g *GrafanaLive) ClientCount(orgID int64, channel string) (int, error) {
 	return len(p.Presence), nil
 }
 
+func (g *GrafanaLive) UsageStats() UsageStats {
+	clients := g.node.Hub().NumClients()
+	users := g.node.Hub().NumClients()
```

Comment:
```suggestion
	users := g.node.Hub().NumUsers()
```

Reply:
ah that explains why these where same! LOL, :facepalm: